backend/src/main/java/gov/cdc/usds/simplereport/service/OrganizationService.java

```diff
@@ -54,6 +59,9 @@ public OrganizationService(OrganizationRepository repo,
                 .collect(Collectors.toList());
         List<Organization> validOrgs = _repo.findAllByExternalId(candidateExternalIds);
         if (validOrgs == null || validOrgs.size() != 1) {
+            Integer numOrgs = (validOrgs == null) ? 0
+                                                  : validOrgs.size();
+            LOG.debug("Found {} organizations for user", numOrgs);
```

Comment:
This should be very rare and is an error condition, so I think we should actually log it on WARN or higher.